entry-test/coverByDominoes.js

```diff
@@ -0,0 +1,10 @@
+'use.strict';
+
+function coverByDominoes(boardHeight, boardWidth){
+
+    let boardSquare = boardHeight * boardWidth;
+
+    return Math.floor(boardSquare / 2);
+};
```

Comment:
http://codeforces.com/problemset/problem/50/A

Size of dominoes are standart - 2x1